index.js

```diff
@@ -1,10 +1,187 @@
-const http = require('http')
 
-const app = http.createServer((request, response) => {
-    response.writeHead(200, { 'Content-Type': 'text/plain' })
-    response.end('Hello World')
+require('dotenv').config()
+const express = require('express')
+const app = express()
+const bodyParser = require('body-parser')
+const morgan = require('morgan')
+const cors = require('cors')
+const Person = require('./models/person')
+
+
+// tad kā? express.json() un/vai bodyParser? atkāpties un noskaidrot
+app.use(express.json())
+app.use(bodyParser.json())
```

Comment:
You shall know the truth; and the truth shall set you free: https://expressjs.com/en/api.html#express.json:~:text=JSON%20payloads%20and-,is%20based%20on%20body%2Dparser.